avocado/core/loader.py

```diff
@@ -795,4 +797,34 @@ def get_decorator_mapping():
         return {test.ExternalRunnerTest: output.TERM_SUPPORT.healthy_str}
 
 
+class RemoteLoader(TestLoader):
+
+    """
+    Remote-runner loader class
+    """
+    name = 'remote'
+
+    def __init__(self, args, extra_params):
+        super(RemoteLoader, self).__init__(args, extra_params)
+        loaders = getattr(args, 'loaders', None)
+        if loaders is not None and 'remote' in loaders:
+            self.remote = True
+        else:
+            self.remote = False
+
+    def discover(self, url, which_tests=DEFAULT):
```

Comment:
This is consistent with the previous suggestion I gave.  What I think brings a bit of misunderstanding is that the this is *not really* a `RemoteLoader`, in the sense that it will attempt to load (resolve) tests remotely.  IMHO it would make it much more clear to name it `DummyLoader`, so that we know that the test loading (resolution) is not (really) being performed.